modules/widgets/facebook-likebox.php

```diff
@@ -207,17 +211,36 @@ function form( $instance ) {
 			</label>
 		</p>
 
+		<p>
+			<label for="<?php echo esc_attr( $this->get_field_id( 'hide_cta' ) ); ?>">
+				<input type="checkbox" name="<?php echo esc_attr( $this->get_field_name( 'hide_cta' ) ); ?>" id="<?php echo esc_attr( $this->get_field_id( 'hide_cta' ) ); ?>" <?php checked( $like_args['hide_cta'] ); ?> />
+				<?php _e( 'Hide Call To Action Button', 'jetpack' ); ?>
```

Comment:
I would escape this:

```suggestion
				<?php esc_html_e( 'Hide Call To Action Button', 'jetpack' ); ?>
```

Same below.